### PR TITLE
Add deployment scenarios section to document two primary use cases for signature-based SRI

### PR DESCRIPTION
**Pull request by ddworken**
As suggested on #9 

Also, I'm not sure why index.html has quite so many changes, I used the latest version of bikeshed to update it. Let me know if there is something else I should be doing here.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -5,9 +5,11 @@
   <title>Signature-based Integrity</title>
   <meta content="w3c/UD" name="w3c-status">
   <link href="https://www.w3.org/StyleSheets/TR/2021/W3C-UD" rel="stylesheet">
-  <meta content="Bikeshed version 82ce88815, updated Thu Sep 7 16:33:55 2023 -0700" name="generator">
+  <meta content="Bikeshed version 6270e4735, updated Tue Aug 6 12:12:30 2024 -0700" name="generator">
```
[the remark this answers is not in the document]
Comment (ddworken) [reply index.html:8]: It looks like you're still using a version of bikeshed from September 2023, which I think is the root cause here. I tried generating this file with curl too and got the same results as with my local version of bikeshed. So I think the large diff here is WAI to update the html file to be generated with the newest version of bikeshed.

**Comment by mikewest** [comment]
I updated `index.html` in a previous commit this morning that I think resolves the discrepancy (and changed the entire template since we're now in WICG). Can you either run the `.html` updates again, or drop them entirely and I'll take care of it in a subsequent CL?

(Really, we should set up to generate a GitHub Pages branch. I'll figure out how to take care of that.)

**Comment by ddworken** [comment]
Done, thanks!

**Comment by mikewest** [comment]
Thanks! Apologies for the back and forth.

**Pull request merged**